lib/xboxrt/libc_extensions/stdlib_ext_.h

```diff
@@ -35,3 +35,29 @@ int mbtowc (wchar_t *pwc, const char *string, size_t n);
 #ifdef __cplusplus
 }
 #endif
+
+
+#ifdef __cplusplus
+// libc++ expects MSVCRT to provide these overloads
+
+inline static long abs (long __x) noexcept
```

Comment:
`noexcept` requires C++11. I can see myself compiling C++98 code though.
It's my understanding that these overloads were also added at specific C++ versions. So this needs standard version checks and potentially dropping the noexpect or using a compiler extension for older versions.

Reply:
I added some preprocessor code to use the old way (`throw()`) if not using at least C++11.
Keep in mind that libc++ headers are only compatible to language version 11 or higher, so if the code actually requires to be built with a lower language version you'll hit incompatibilities.